src/app/index.tsx

```diff
@@ -70,10 +73,24 @@ export default (options: Options) => {
     reducer,
   });
 
+  const language = (typeof window === 'undefined') ? 'en-us' : window.navigator.language;
+  addLocaleData([...en, ...cs]);
+
+  const messages = new Map()
+  messages.set('en', { 'i18n:404': 'page not found' });
+  messages.set('cs', { 'i18n:404': 'požadovaná stránka neexistuje'});
```

Comment:
we may want to hold off on the second language file, that will definitely slow us down